picard/script.py

```diff
@@ -1146,3 +1146,23 @@ def func_map(parser, multi, loop_code, separator=MULTI_VALUED_JOINER):
     if not isinstance(separator, str):
         separator = separator.eval(parser)
     return separator.join(multi_value)
+
+
+@script_function(eval_args=False)
+def func_joinmulti(parser, multi, join_phrase, separator=MULTI_VALUED_JOINER):
```

Comment:
I would be in favor of using `$join` as the function name here.

Reply:
I'm in favor of `$join` too